test/EventListener/CacheControlSubscriberTest.php

```diff
@@ -75,7 +75,7 @@ public function it_does_not_change_for_streamed_responses()
      */
     public function it_does_not_change_for_already_not_modified_responses()
     {
-        $subscriber = new CacheControlSubscriber();
+        $subscriber = new CacheControlSubscriber(1, 2, 3);
```

Comment:
not perfectly clear, but language limitations unlike `kwargs`

Reply:
Thinking might be better to just set a string. Would allow for things like `no-store` in some environments, a maybe `s-maxage` in `prod`, without having lots of parameters.